src/app/generate/generate_test.go

```diff
@@ -396,6 +396,64 @@ dependencies:
         commit: chore(deps): bump anotherdep from 0.0.1 to 0.0.2 (#69)
 			`) + "\n",
 		},
+		{
+			name:   "Markdown_Renovate_Filter_ExcludedDependencies",
+			md:     mdChangelog,
+			args:   fmt.Sprintf("--dependabot=false --excluded-dependencies=%s", path.Join("..", "testdata", "excluded-dependencies.yml")),
+			author: "renovate[bot] <renovatebot@imadethisup.com>",
+			commits: []string{
+				"chore(deps): update github.com/stretchr/testify to 1.8.0",
+				"chore(deps): update helm release common-library to v1.0.4 (#401)",
+			},
+			// Note: meta.commit is actually the commit hash.
+			// As it is nontrivial to know the commit hash in advance, to make tests easier to write, test writers
+			// should specify the commit message instead. This test will replace it with the actual hash in runtime.
+			expected: strings.TrimSpace(`
+notes: |-
+    ### Important announcement (note)
+    This is a release note
+changes:
+    - type: breaking
+      message: Support has been removed
+    - type: security
+      message: Fixed a security issue that leaked all data
+dependencies:
+    - name: common-library
+      to: v1.0.4
+      meta:
+        pr: "401"
+        commit: chore(deps): update helm release common-library to v1.0.4 (#401)
+			`) + "\n",
+		},
+		{
+			name:   "Markdown_Dependabot_Filter_ExcludedDependencies",
+			md:     mdChangelog,
+			args:   fmt.Sprintf("--renovate=false --excluded-dependencies=%s", path.Join("..", "testdata", "excluded-dependencies.yml")),
+			author: "dependabot <dependabot@github.com>",
+			commits: []string{
+				"chore(deps): bump github.com/stretchr/testify from 1.7.0 to 1.8.0",
+				"chore(deps): bump thisdep from 1.7.0 to 1.10.1",
+			},
+			// Note: meta.commit is actually the commit hash.
```

Comment:
does it make sense to copy this message each time?

Reply:
Probably not, since it was in the previous 10 subtests I didn't want to "break with the standard" 😁 I'm going to remove it from all of them except the 1st.

Reply:
Done in 4ffdd63741f6aca325bc28144f13abf165546015.